xsuite/src/proxy/sproxy.ts

```diff
@@ -18,10 +18,18 @@ export class SProxy extends Proxy {
     return Proxy.fetch(`${baseUrl}/admin/set-current-block`, block);
   }
 
+  static setPreviousBlock(baseUrl: string, block: Block) {
```

Comment:
Move this code below `setCurrentBlock` in order to have:

- `static setCurrentBlock`
- `setCurrentBlock`
- `static setPreviousBlock`
- `setPreviousBlock`